src/method/method_query_handler.cpp

```diff
@@ -190,6 +190,16 @@ namespace cubmethod
     return error;
   }
 
+  int
+  query_handler::prepare_retry ()
+  {
+    if (is_prepared ())
```

Comment:
is_prepared means "was prepared" ?

Reply:
In the query handler, is_prepared() means that the statement handler is created by calling `db_compile_statement()`.

Reply:
To be clear, the meaning of is_prepared here does not test whether XASL exists on the server side. It means whether the internal structure including PT_NODE was created by parsing the query received from the SP. Therefore, in this function, "is prepared" is checked, not "was prepared".

Reply:
Ok, I understood.